frameworks/g-cloud-9/questions/services/virtualisationTechnologiesUsed.yml

```diff
@@ -0,0 +1,29 @@
+name: "Virtualisation technologies used"
+question: "What virtualisation technologies are used?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+
+list_item_name: "virtualisation technologies"
+type: list
+
+validations:
+  -
+    name: answer_required
```

Comment:
Surely this should be an optional question?  Unless you want to capture `N/A`, `None`, `We don't use virtualisation`, etc. in the services.